cmd/plugin/builder/template/plugintemplates/common.mk.tmpl

```diff
@@ -1,6 +1,6 @@
 # Ensure Make is run with bash shell as some syntax below is bash-specific
-SHELL:=bash
-.ONESHELL:
```

Comment:
This was unnecessary and was causing problems due to  `SHELL := /usr/bin/env bash` that used to be in the `plugin-tooling.mk` file.

My guess is that the `.ONESHELL` was added as a convenience to plugin writers as it can make writing makefile targets easier.
However this can turn out to be confusing for people who don't usually use that setting when writing makefiles.

So it was better to remove it in both those perspectives (simplification and fix).